source/FreeRTOS_Sockets.c

```diff
@@ -3139,7 +3139,18 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t * pxSocket )
                 }
 
                 /* Go sleeping until we get any down-stream event */
-                ( void ) xEventGroupWaitBits( pxSocket->xEventGroup, ( EventBits_t ) eSOCKET_CONNECT, pdTRUE /*xClearOnExit*/, pdFALSE /*xWaitAllBits*/, xRemainingTime );
+                EventBits_t uxEvents = xEventGroupWaitBits( pxSocket->xEventGroup,
```

Comment:
Maybe declare the variable at top of the block? Maybe just after `for( ; ; )`?

Reply:
Done so, thanks